docker/wrapper.py

```diff
@@ -302,13 +447,23 @@ def main():
 
     #print("star {} kallisto {} rsem {} args".format(args.star, args.kallisto, args.rsem))
     #Input for star and rsem will be empty if user wants to run kallisto only so test for not x
-    require(all( (x.startswith('/') or not x) for x in [args.star, args.kallisto, args.rsem]),
+
+    #file paths should start with /, file://, http://, s3://, or ftp://
+    if args.auto_scale:
```

Comment:
2nd `if args.auto_scale`, consolidate if possible.

Reply:
will put in a later release